src/integration-test/java/io/aiven/kafka/connect/gcs/IntegrationTest.java

```diff
@@ -162,11 +159,13 @@ final void tearDown() {
         connectRunner.awaitStop();
     }
 
-    @Test
-    final void basicTest() throws ExecutionException, InterruptedException {
+    @ParameterizedTest
+    @ValueSource(strings = {"none", "gzip", "snappy", "zstd"})
+    final void basicTest(final String compression) throws ExecutionException, InterruptedException {
+        final CompressionType compressionType = CompressionType.forName(compression);
```

Comment:
Seems not needed

Reply:
I have left only "none" compression. Is it ok?